test/unit/models/local_transaction_services_test.rb

```diff
@@ -10,4 +10,10 @@ class LocalTransactionServicesTest < ActiveSupport::TestCase
       assert_not LocalTransactionServices.instance.unavailable?(461, "Northern Ireland")
     end
   end
+
+  context ".content" do
+    should "return a string with the given country and local authority name in" do
+      assert_equal "This service is unavailable in Dundee, Scotland", LocalTransactionServices.instance.content(461, "Scotland", "Dundee")
+    end
+  end
```

Comment:
Maybe with having a test for what happens if content is missing from the config. I.e. a test showing how we get an empty string back

Reply:
Ah, yes - good point thanks.  We should also account for no content at all, but with the key; and no content key.  Tests added.